src/Extensibility/Directory.Build.props

```
@@ -0,0 +1,7 @@
+<!-- Directory.Build.props for Extensibility -->
+<Project>
+
+    <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
```

Technically correct, but in this case the same as the simpler:
```suggestion
    <Import Project="$(MSBuildThisFileDirectory)../Directory.Build.props" />
```

---

Done. I chose flexibility (in moving directories up and down the hierarchy) over simplicity. But in retrospect, that's a good choice if the Import is in a `.csproj` file, but not so much when in a `.props` file.